project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=0.13.17
+sbt.version=1.2.8
```

Comment:
How did I not notice that 0.13.17 is older than 1.2.8?

🤦‍♀

Reply:
I think the 0.13 branch is forked from the 1.x branch and still maintained, so it's not clear if it is actually older. The `0.13` fork is the "old" one though.